src/helpers/helpers.canvas.js

```diff
@@ -146,22 +132,18 @@ var exports = module.exports = {
 			ctx.lineTo(xOffset, yOffset);
 			ctx.moveTo(-xOffset, yOffset);
 			ctx.lineTo(xOffset, -yOffset);
-			ctx.closePath();
 			break;
 		case 'line':
-			ctx.beginPath();
 			ctx.moveTo(-radius, 0);
 			ctx.lineTo(radius, 0);
-			ctx.closePath();
 			break;
 		case 'dash':
-			ctx.beginPath();
 			ctx.moveTo(0, 0);
 			ctx.lineTo(radius, 0);
-			ctx.closePath();
 			break;
 		}
 
+		ctx.fill();
 		ctx.stroke();
 		ctx.restore();
```

Comment:
Is there any impact (in the calling code) to not call `ctx.closePath();` anymore before returning?